api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -80,7 +80,7 @@
 
         <field xml.transient="true">
           <name>pomFile</name>
-          <version>5.0.0+</version>
+          <version>4.2.0</version>
```

Comment:
was "5.0.0+" and now is only "4.2.0", did you miss "+"?

Reply:
Yes, I fixed those.